services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/api_client/_base.py

```diff
@@ -68,11 +65,11 @@ async def request_wrapper(zelf: "BaseThinClient", *args, **kwargs) -> Response:
         # pylint: disable=protected-access
         try:
             async for attempt in AsyncRetrying(
-                stop=stop_after_attempt(zelf._request_max_retries),
+                stop=stop_after_delay(zelf.request_timeout),
```

Comment:
Side effect of this timeout:
- service has an extra (60-7) second delay before starting (there are issues with the logic of the health check event)

Reply:
- requires https://github.com/ITISFoundation/osparc-simcore/pull/3586